docs/tools/buildmodref.py

```diff
@@ -36,7 +35,7 @@ def writeapi(package, outdir, source_version, other_defines=True):
     # are not (re)generated. This avoids automatic generation of documentation
     # for older or newer versions if such versions are installed on the system.
 
-    installed_version = V(module.__version__)
+    installed_version = Version(module.__version__)
     if source_version != installed_version:
```

Comment:
Actually, I think this is just an old holdover from pre-autodoc Sphinx that was in nipype. I think we should just drop the file altogether.

Reply:
Yeah, seems safe.